backend/src/main/java/vn/edu/hust/nmcnpm_20242_n3/entity/Permission.java

```diff
@@ -10,4 +15,6 @@ public class Permission {
     int id;
     @Column(unique = true, nullable = false)
     String name;
+    @ManyToMany(mappedBy = "permissions")
+    private Set<Role> roles;
```

Comment:
I don't think we will need this, as we already have a table for the relationship between `Permission` and `Role`. This will create an infinite recursive response too. Should we consider removing this?